### PR TITLE
 Language: Make production error message translatable (replaces #6197)

### PR DESCRIPTION
**Pull request by sba**
replaces #6197

With this pull request the error message in the file app/Views/errors/html/production.php can be translated.

Translation for error_404.php and error_exception.php will follow with a other PR

### CODE REVIEW

**Comment by sba** [comment]
@kenjis once again with a new, clean PR!
...I have no idea why some tests now were not successful..

**Comment by sba** [comment]
> Why is "system/Language/en/Errors.php" the same as "app/Views/errors/html/production.php"?

ahh, crap! Copy/paste error - will fix it!

File: app/Views/errors/html/production.php [hunk 1]
```diff
@@ -4,21 +4,21 @@
...
+    <title><?= lang('Errors.whoops') ?></title>
 
     <style type="text/css">
         <?= preg_replace('#[\r\n\t ]+#', ' ', file_get_contents(__DIR__ . DIRECTORY_SEPARATOR . 'debug.css')) ?>
     </style>
 </head>
 <body>
 
-    <div class="container text-center">
+<div class="container text-center">
```
Comment (kenjis) [inline app/Views/errors/html/production.php:15]: Please revert the indentation.
Or do you have any reason to change the indentation?

**Review by kenjis — APPROVED** [review]
LGTM!
Thank you.

**Review by paulbalandan — APPROVED** [review]

**Review by MGatner — APPROVED** [review]

**Pull request merged**